library/build.go

```diff
@@ -542,6 +547,22 @@ func (b *Build) GetDistribution() string {
 	return *b.Distribution
 }
 
+func (b *Build) GetPusherName() string {
+	if b == nil || b.PusherName == nil {
+		return ""
+	}
+
+	return *b.PusherName
+}
+
+func (b *Build) GetPusherEmail() string {
+	if b == nil || b.PusherEmail == nil {
+		return ""
+	}
+
+	return *b.PusherEmail
+}
```

Comment:
Looks like comments are missing here.

To show an example of what those comments may look like:

https://github.com/go-vela/types/blob/911ffbea2e427883ccca2cc3f3769fdae75e0946/library/build.go#L428-L439